Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/BackLighting.azsli

```diff
@@ -16,24 +16,27 @@
 #include <Atom/Features/PBR/LightingOptions.azsli>
 
 // Analytical integation (approximation) of diffusion profile over radius, could be replaced by other pre integrated kernels
-// such as sum of Gaussian
+// such as sum of Gaussian (see T(s))
 float3 TransmissionKernel(float t, float3 s)
 {
     float3 exponent = s * t;
     return 0.25 * (1.0 / exp(exponent) + 3.0 / exp(exponent / 3.0));
 }
 
-float ThinObjectFalloff(const float3 surfaceNormal, const float3 dirToLight)
+// [specific profile for SKIN (not used ATM, replaced by TransmissionKernel)]
+// Analytical integation (approximation) of diffusion profile over radius, could be precomputed in a LUT
```

Comment:
Could you add reference for that approximation??

Reply:
Done in https://github.com/o3de/o3de/pull/6428/commits/7052b99671027ceb36174d95e275d935eeb5be42. The reference is [GPU Gems 3 - 14. Advanced Techniques for Realistic Real-Time Skin Rendering](https://developer.nvidia.com/gpugems/gpugems3/part-iii-rendering/chapter-14-advanced-techniques-realistic-real-time-skin) (section 14.4.7)